### PR TITLE
OBSDOCS-181: Alertmanager should use the proxy environment variables

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `enterprise-4.18` and `monitoring-docs-restructure`

Issue: [OBSDOCS-181](https://issues.redhat.com/browse/OBSDOCS-181)

Link to docs preview:  https://85113--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/managing-alerts.html#configuring-notifications-for-default-platform-alerts_managing-alerts

QE review:
- [x] QE has approved this change.

**Additional information:**

### CODE REVIEW

**Review by simonpasquier — COMMENTED** [review]
/lgtm

File: modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc [hunk 1]
```diff
@@ -42,37 +42,40 @@ $ oc -n openshift-monitoring get secret alertmanager-main --template='{{ index .
...
 - name: default
 - name: watchdog
 - name: <receiver>
-  <receiver_configuration> #<7>
+  <receiver_configuration> # <8>
 ----
-<1> Specify how long Alertmanager waits while collecting initial alerts for a group of alerts before sending a notification.
-<2> Specify how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
-<3> Specify the minimum amount of time that must pass before an alert notification is repeated.
+<1> If you configured an HTTP cluster-wide proxy, you can set the `proxy_from_environment` parameter to `true` to enable proxying for all alert receivers.
```
Comment (juzhao) [inline modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc:69]:
```suggestion
<1> If you configured an HTTP cluster-wide proxy, set the `proxy_from_environment` parameter to `true` to enable proxying for all alert receivers.
```
from same description in https://github.com/openshift/openshift-docs/pull/85113/files#diff-9d3a4011173601a722946389c2d8bd36d09c69420017ae76e66ae45fe5e501beR50

**Comment by aspauldi** [comment]
LGTM! 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Review by xenolinux — COMMENTED** [review]
Merge review LGTM

**Pull request merged**